### ISSUE TITLE
博主，mask rcnn项目中的test.jpg去哪里了

### ISSUE THREAD

**Issue by stevenk12** [WZMIAOMIAO/deep-learning-for-image-processing#590]
**System information**
* Have I written custom code:
* OS Platform(e.g., window10 or Linux Ubuntu 16.04):
* Python version:
* Deep learning framework and version(e.g., Tensorflow2.1 or Pytorch1.3):
* Use GPU or not:
* CUDA/cuDNN version(if you use GPU):
* The network you trained(e.g., Resnet34 network):

**Describe the current behavior**

**Error info / logs**

**Comment by WZMIAOMIAO** [comment]
自己随便找张图片就行了。
